src/SonarScanner.MSBuild.Common/TFS/BuildSettings.cs

```diff
@@ -239,48 +213,19 @@ public string CoverageToolUserSuppliedPath
             private set;
         }
 
-        #endregion Public properties
-
-        #region Public calculated properties
+        public string SonarConfigDirectory => Path.Combine(AnalysisBaseDirectory, "conf");
```

Comment:
NIT: The convention is like so for non-trivial accessors.
```suggestion
        public string SonarConfigDirectory =>
            Path.Combine(AnalysisBaseDirectory, "conf");
```